src/components/Breadcrumbs/Breadcrumbs.scss

```diff
@@ -0,0 +1,16 @@
+.breadcrumb {
+  margin-top: 6vh;
```

Comment:
Could this be converted to `rem` or `px`, `vh` is relative to the screen size which we probably don't want

Reply:
I just left the styles as I found them, but looking at the figma board now, it looks like the breadcrumbs should be a little higher than they are. If I remove the margin-top it seems to look about even with the sidebar. Should I maybe just remove this line in that case?